cmd/skywire-cli/commands/node/gen-config.go

```diff
@@ -23,12 +23,14 @@ var (
 	output        string
 	replace       bool
 	configLocType = pathutil.WorkingDirLoc
+	testenv bool
```

Comment:
Could you please add more spaces between `testenv` and `bool` to match the indentation of `=` above?